### PR TITLE
번역: AlexNet 번역

### PR DESCRIPTION
**Pull request by JDhyeok**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #46 
 
## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._
AlexNet 영한 번역

### CODE REVIEW

File: pytorch_vision_alexnet.md [hunk 1]
```diff
@@ -50,47 +49,49 @@ preprocess = transforms.Compose([
     transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
 ])
 input_tensor = preprocess(input_image)
-input_batch = input_tensor.unsqueeze(0) # create a mini-batch as expected by the model
+input_batch = input_tensor.unsqueeze(0) # 모델에서 요구하는 형식인 mini batch 생성
```
Comment (cepiloth) [inline pytorch_vision_alexnet.md:52]: 수고하셨습니다. 미니 배치는 음차표기 그대로 하는것을 추천드려요.
https://github.com/PyTorchKorea/tutorials-kr/blob/master/TRANSLATION_GUIDE.md

**Review by cepiloth — COMMENTED** [review]
번역하느라고 수고 하셨습니다.

File: pytorch_vision_alexnet.md [hunk 2]
```diff
@@ -23,23 +23,22 @@ model = torch.hub.load('pytorch/vision:v0.10.0', 'alexnet', pretrained=True)
 model.eval()
 ```
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(3 x H x W)`, where `H` and `W` are expected to be at least `224`.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
+모든 사전 훈련된 모델들은 같은 방법으로 정규화된 입력 이미지를 입력받습니다. 예시로, `(3 x H x W)`의 3-채널 RGB 이미지들의 미니 배치일 때 `H`와 `W`의 크기는 224 이상이어야 합니다.
```
Comment (HyunKyungHan) [inline pytorch_vision_alexnet.md:26]: 음..이건 그냥 제 생각인데 이 문단의 1-2번째 줄이 재훈님 FCN 번역 1-2번째 줄이랑 거의 비슷해서 번역 내용을 통일해도 좋을 것 같아요! 
다음 팀 회의 때 이야기해봐도 좋을 것 같네요 :)

**Review by HyunKyungHan — COMMENTED** [review]
번역 수고하셨습니다!!

**Comment by codingbowoo** [comment]
@JDhyeok 님, 팀원분들 코멘트 반영해주시면 리뷰 및 머지 진행하겠습니다.

File: pytorch_vision_alexnet.md (hunk 2 above)
Comment (cepiloth) [reply pytorch_vision_alexnet.md:26]: 그러네요 완전히 같아요. 통합해도 될거같아요 :)

File: pytorch_vision_alexnet.md (hunk 1 above)
Comment (JDhyeok) [reply pytorch_vision_alexnet.md]: 리뷰 감사합니다. 반영 완료했습니다.

File: pytorch_vision_alexnet.md (hunk 2 above)
Comment (JDhyeok) [reply pytorch_vision_alexnet.md]: 리뷰 감사합니다. 반영 완료했습니다.

**Review by Taeyoung96 — APPROVED** [review]
깔끔한 번역 감사드립니다.

File: pytorch_vision_alexnet.md [hunk 3]
```diff
@@ -50,47 +49,49 @@ preprocess = transforms.Compose([
...
 top5_prob, top5_catid = torch.topk(probabilities, 5)
 for i in range(top5_prob.size(0)):
     print(categories[top5_catid[i]], top5_prob[i].item())
 ```
 
-### Model Description
+### 모델 설명
 
-AlexNet competed in the ImageNet Large Scale Visual Recognition Challenge on September 30, 2012. The network achieved a top-5 error of 15.3%, more than 10.8 percentage points lower than that of the runner up. The original paper's primary result was that the depth of the model was essential for its high performance, which was computationally expensive, but made feasible due to the utilization of graphics processing units (GPUs) during training.
+AlexNet은 2012년도 ImageNet Large Scale Visual Recognition Challnege (ILSVRC)에 참여한 모델입니다. 이 네트워크는 15.3%의 top-5 에러율을 달성했고, 이는 2위보다 10.8%P 낮은 수치입니다. 원 논문의 주요 결론은 높은 성능을 위해 모델의 깊이가 필수적이라는 것이었습니다. 이는 계산 비용이 많이 들지만, 학습 과정에서 GPU의 사용으로 가능해졌습니다.
```
Comment (codingbowoo) [inline pytorch_vision_alexnet.md:87]:
```suggestion
AlexNet은 2012년도 ImageNet Large Scale Visual Recognition Challenge (ILSVRC)에 참여한 모델입니다. 이 네트워크는 15.3%의 top-5 에러율을 달성했고, 이는 2위보다 10.8%P 낮은 수치입니다. 원 논문의 주요 결론은 높은 성능을 위해 모델의 깊이가 필수적이라는 것이었습니다. 이는 계산 비용이 많이 들지만, 학습 과정에서 GPU의 사용으로 가능해졌습니다.
```
오탈자 수정 부탁드립니다! 
그와 별개로, ILSVRC 라고 명시해주신 점이 좋네요. 👍

**Review by codingbowoo — CHANGES_REQUESTED** [review]
타이포만 수정해주시면 바로 머지 진행하겠습니다!
깔끔한 번역 감사드립니다. :+1:

File: pytorch_vision_alexnet.md (hunk 3 above)
Comment (JDhyeok) [reply pytorch_vision_alexnet.md]: 한국어 맞춤법만 확인하고 영어를 깜빡했네요!
수정했습니다. :)

**Review by codingbowoo — APPROVED** [review]
LGTM! 수고 많으셨습니다!
